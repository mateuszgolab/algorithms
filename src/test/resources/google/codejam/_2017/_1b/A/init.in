9
2525 1
2400 5
300 2
120 60
60 90
100 2
80 100
70 10
1000000000 1
999999999 1
1000000000 2
2 2
1 3
2 1
1 10000
200000 2
5000 6
5001 1
587348605 2
538167563 8690
570189796 3630
1000000000 2
999999999 2
999999998 3