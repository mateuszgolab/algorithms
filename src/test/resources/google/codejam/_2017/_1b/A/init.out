Case #1: 101.000000
Case #2: 100.000000
Case #3: 33.333333
Case #4: 1000000000.000000
Case #5: 2.000000
Case #6: 20000.000000
Case #7: 1.025646
Case #8: 103781.033705
Case #9: 1500000000.000000
